utils/utils_supported.go

```diff
@@ -101,6 +101,20 @@ func GetCgroupProcess(pid int) (string, error) {
 	return getCgroupProcess(fmt.Sprintf("/proc/%d/cgroup", pid))
 }
 
+// CheckifSystemdSessionisValid checks if valid user session ever existsted or not.
+func CheckifSystemdSessionisValid(uid int) bool {
+	slicePath := fmt.Sprintf("/sys/fs/cgroup/user.slice/user-%d.slice/user@%d.service", uid, uid)
```

Comment:
does systemd document that it uses always this path and it won't be changed in future?

Reply:
Should we define this in containers.conf just in case?

Reply:
@giuseppe @TomSweeneyRedHat I checked so systemd doesn't guarantee about this path, although its accepted as a norm in most of cases but in some cases it could also be `/sys/fs/cgroup/user.slice/user-1000.slice/session-11.scope`. I'll refactor the code and get session details from Dbus instead.

Reply:
@giuseppe @TomSweeneyRedHat Made changes instead of doing a `stat` on a static path we are now verifying valid systemd session for rootless user using d-bus interface of `systemd-logind` following method is guaranteed by systemd.